newIDE/app/src/Utils/GDevelopServices/Play.js

```diff
@@ -208,6 +208,35 @@ export const createLeaderboard = async (
   return response.data;
 };
 
+type DuplicateLeaderboardBody = {|
```

Comment:
To be consistent with `LeaderboardUpdatePayload`, you could use `LeaderboardDuplicationPayload` instead